### PR TITLE
remove rtk documentation

### PR DESCRIPTION
**Pull request by samuelbartels20**
Please review

### CODE REVIEW

File: content/en/docs/v3.4/op-guide/container.md [hunk 1]
```diff
@@ -1,72 +1,14 @@
...
-Verify the cluster is healthy and can be reached.
-
-```
-ETCDCTL_API=3 etcdctl --endpoints=http://172.16.28.21:2379,http://172.16.28.22:2379,http://172.16.28.23:2379 endpoint health
-```
 
 ### DNS
 
-Production clusters which refer to peers by DNS name known to the local resolver must mount the [host's DNS configuration](https://coreos.com/kubernetes/docs/latest/kubelet-wrapper.html#customizing-rkt-options).
+Production clusters which refer to peers by DNS name known to the local resolver must mount the [host&#39;s DNS configuration](https://coreos.com/kubernetes/docs/latest/kubelet-wrapper.html#customizing-rkt-options).
```
Comment (jmhbnz) [inline content/en/docs/v3.4/op-guide/container.md:11]: Let's also remove this line from each of the three files as it looks to be rkt specific.

Comment (jmhbnz) [reply content/en/docs/v3.4/op-guide/container.md]: Thanks @samuelbartels20 - Can you please squash commits so we are ready to merge, i.e.

```bash
git reset --soft HEAD~2
git commit -m "Removed rkt documentation."
git push --force
```

Comment (samuelbartels20) [reply content/en/docs/v3.4/op-guide/container.md]: > Thanks @samuelbartels20 - Can you please squash commits so we are ready to merge, i.e.
> 
> ```shell
> git reset --soft HEAD~2
> git commit -m "Removed rkt documentation."
> git push --force
> ```

@jmhbnz Please i have made the changes

**Pull request merged**